tfhe/src/c_api/shortint/parameters.rs

```diff
@@ -398,6 +410,8 @@ expose_as_shortint_pbs_parameters!(
     PARAM_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M64,
 );
 
+// TODO: make API using this type
+#[allow(dead_code)]
 pub struct CompressionParameters(
     pub(crate) crate::shortint::parameters::list_compression::CompressionParameters,
 );
```

Comment:
no configuration in the HL API to enable compression ?

Reply:
Actually yes, it's used by `config_builder_enable_compression`
It appeared unused because I didn't have `high-level-c-api` feature enabled
Removed